==============
 Introduction
==============

*Text shamelessly stolen from the Nucleic Acids Res (2017) 45 (D1):
D535-D542.*


The Pathosystems Resource Integration Center (PATRIC) is the
all-bacterial Bioinformatics Resource Center (BRC)
(https://www.patricbrc.org). Established by the National Institute
of Allergy and Infectious Diseases (NIAID), PATRIC provides
researchers with an online resource that stores and integrates a
variety of data types—e.g. genomics, transcriptomics, protein–protein
interactions, 3D protein structures and sequence typing data, and
associated metadata. The PATRIC website is primarily organism-centric,
with various levels of genomic data and associated information related
to each included organism. While the PATRIC homepage highlights the 22
NIAID Category A–C genera for easy access to data associated with many
pathogenic species, PATRIC's compilation and organization of all
relevant data for all available sequenced bacteria are standardized
according to bacterial (NCBI) taxonomy, with options for viewing
sets of genomes within the hierarchical bacterial tree.

With an emphasis on consistency in comparative genomic analysis,
PATRIC has standardized annotations for over 80,000 bacterial genomes
(as of August 2016) using the Rapid Annotation using Subsystems
Technology toolkit (RASTtk). This number includes more than 6000
genomes from antibiotic resistance studies that whose reads were
obtained from the Sequence Read Archive (SRA) and then assembled,
annotated and made public in PATRIC. It also includes nearly 500
genomes submitted to PATRIC by the United States Agricultural
Department (USDA) that were part of their pathogen monitoring
process. In addition to the RASTtk-based annotations, PATRIC preserves
and provides the legacy annotations from GenBank and RefSeq,
allowing researchers to contrast differences between the
two. Summaries of the different data types (e.g. genes, RNAs, etc.)
from both PATRIC and RefSeq annotations are available across different
taxonomic levels, and are also provided for both genomes and for
individual genes.

The PATRIC website  is designed to enhance the user
experience. All data types are now indexed with SOLR and interfaced
with the PATRIC Data API in order to achieve the user interface goals
of providing access to large, complex datasets, both public and
private. This access allows for searching, displaying, filtering and
downloading genomes, genomic feature, specialty genes, protein
families, pathways and experiment data. Easy access to tools,
services and the workspace are provided on the PATRIC home page.
In addition, integrated views of user-selected groups are available.

The PATRIC site incorporates a set of services that are designed to
streamline and simplify common bioinformatic workflows. These services
include genome assembly, genome annotation, RNA-Seq analysis,
expression import, proteome comparison, metabolic model reconstruction
and variation analysis. All of these services are available through a
private workspace that allows users to compare their private data to
the public data in the PATRIC database. Researchers may upload their
data files (e.g. sequence reads or assembled genomes), run the desired
analysis services, integrate their private data with the data in
PATRIC for comparative analysis, store the resulting output files and
download the results.


*I would like to include some words about who the primary users of
PATRIC are and what sort of work they do. I'd also like to include
some concrete examples that we can link to the detailed tutorials to
allow readers to recreate the examples. Ideally we'd have at least one
meaty example that relates to published work, and perhaps some easier
ones as well.*


